### PR TITLE
Text tweak “find a consultant”

### PR DESCRIPTION
**Pull request by reinhardt**
syslabcom/scrum#1285

### CODE REVIEW

File: src/euphorie/deployment/locales/de/LC_MESSAGES/euphorie.po [hunk 1]
```diff
@@ -2199,6 +2199,11 @@ msgstr ""
 msgid "description_measures_text_handling"
 msgstr ""
 
+#. Default: "Find an OiRA consultant you want to contact or add to your organisation to provide support."
+#: Euphorie/src/euphorie/client/browser/templates/consultants.pt:44
+msgid "description_oira_consultants"
+msgstr "Finden Sie einen OiRA-Berater, den Sie kontaktieren oder zu Ihrer Organisation hinzufügen möchten, um Unterstützung zu leisten."
```
Comment (thet) [inline src/euphorie/deployment/locales/de/LC_MESSAGES/euphorie.po:2205]: Shouldn't that be:
```suggestion
msgstr "Finden Sie einen OiRA-Berater, den Sie kontaktieren oder zu Ihrer Organisation hinzufügen möchten um Unterstützung zu erhalten."
```

File: src/euphorie/deployment/locales/nl/LC_MESSAGES/euphorie.po [hunk 2]
```diff
@@ -2243,6 +2243,11 @@ msgstr ""
 msgid "description_measures_text_handling"
 msgstr ""
 
+#. Default: "Find an OiRA consultant you want to contact or add to your organisation to provide support."
+#: Euphorie/src/euphorie/client/browser/templates/consultants.pt:44
+msgid "description_oira_consultants"
+msgstr "Ontdek hoe u in contact kunt komen met een OiRA-adviseur op ${email}."
```
Comment (thet) [inline src/euphorie/deployment/locales/nl/LC_MESSAGES/euphorie.po:2249]: I‌ think `${email}` is wrong here. The original msgstr doesn't conain a `i18n:name` part.

**Review by thet — DISMISSED** [review]
Two comments which need to be fixed.
Also, please rebase or otherwise fix the merge conflicts.
Other thant that, LGTM

**Review by thet — CHANGES_REQUESTED** [review]
please see the comments

**Comment by reinhardt** [comment]
> Two comments which need to be fixed. Also, please rebase or otherwise fix the merge conflicts. Other thant that, LGTM

Usually I don't copy translations from prototype for osha because they go through their translation process anyway. This time I thought I'd try it, but apparently it was not a good idea. I've removed them again.

**Review by thet — APPROVED** [review]

**Pull request merged**